lib/src/main/kotlin/AdversarialStatelessSolver.kt

```diff
@@ -1,291 +0,0 @@
-import kotlin.math.ln
```

Comment:
@JunTaoLuo instead of deleting this entirely, could we keep this intact, but allow the user to specify any heuristics / minimax search that would improve on this further in an open-source world?

Reply:
No that wouldn't make sense. Instead we should improve Stateless and Stateful solvers so that heuristics can be added.